### PR TITLE
Update text symbol gallery example

### PR DESCRIPTION
**Pull request by michaelgrund**
**Description of proposed changes**

According to a recent [forum post](https://forum.generic-mapping-tools.org/t/changing-symbol-outline-color-using-plot/2410), here's an update for the text symbol gallery example. Since it is not fully clear from the current example how to change the outline color of the symbols, for the last symbol now the outline color is changed and the docstring is slightly adjusted.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Review by seisman — APPROVED** [review]
Looks good to me.

File: examples/gallery/symbols/text_symbols.py [hunk 1]
```diff
@@ -10,10 +10,10 @@
 different characters) and *string* can be a letter or a text string
 (less than 256 characters). Optionally, you can append **+f**\ *font* to
 select a particular font [Default is :gmt-term:`FONT_ANNOT_PRIMARY`] as well as
-**+j**\ *justify* to change the justification [Default is CM]. Outline
-and fill color of the text symbols can be customized via the ``pen``
-and ``color`` parameters, respectively.
-
+**+j**\ *justify* to change the justification [Default is CM]. Outline color of
```
Comment (willschlitzer) [inline examples/gallery/symbols/text_symbols.py:13]:
```suggestion
**+j**\ *justify* to change the justification [Default is CM]. The outline color of
```

File: examples/gallery/symbols/text_symbols.py [hunk 2]
```diff
@@ -10,10 +10,10 @@
...
+the text symbols can also be defined via the ``style`` parameter [Default is
```
Comment (willschlitzer) [inline examples/gallery/symbols/text_symbols.py:14]:
```suggestion
the text symbols can be defined via the ``style`` parameter [Default is
```
Is there another way that the outline color can be set? If not, this shouldn't say "also"

Comment (willschlitzer) [inline examples/gallery/symbols/text_symbols.py:14]:
```suggestion
the text symbols can be defined via the ``style`` parameter [Default is
```
Is there another way that the outline color can be set? If not, this shouldn't say "also"

File: examples/gallery/symbols/text_symbols.py [hunk 3]
```diff
@@ -10,10 +10,10 @@
...
+black] while the fill color and the outline width can be customized via the
+``color`` and ``pen`` parameters, respectively.
```
Comment (willschlitzer) [inline examples/gallery/symbols/text_symbols.py:16]:
```suggestion
black], the fill color can be set with the ``color`` parameter, and the 
outline width can be customized with the ``pen`` parameter.
```
To minimize confusion for non-native English speakers (and maybe some native English speakers), I think it makes it easier to follow along to have the parameter name next to what is being set.

File: examples/gallery/symbols/text_symbols.py [hunk 4]
```diff
@@ -37,7 +37,8 @@
 # color fill is set to "gold"
 fig.plot(x=5.5, y=1.5, style="l3.5c+ts+fTimes-Italic", color="gold", pen=pen)
 # plot the pi symbol (\160 is octal code for pi) of size 3.5c, for this use
-# the "Symbol" font, color fill is set to "magenta4"
-fig.plot(x=7, y=1.5, style="l3.5c+t\160+fSymbol", color="magenta4", pen=pen)
+# the "Symbol" font, outline color of the symbol is set to "darkorange",
+# color fill is set to "magenta4"
```
Comment (willschlitzer) [inline examples/gallery/symbols/text_symbols.py:41]:
```suggestion
# the "Symbol" font, the outline color of the symbol is set to 
# "darkorange", the color fill is set to "magenta4"
```
I think this reads better adding articles in before outline and color fill

File: examples/gallery/symbols/text_symbols.py (hunk 2 above)
Comment (michaelgrund) [reply examples/gallery/symbols/text_symbols.py:14]: I'm not fully sure, the GMT documentations is not really detailed at this point.

File: examples/gallery/symbols/text_symbols.py (hunk 3 above)
Comment (michaelgrund) [reply examples/gallery/symbols/text_symbols.py]: Thanks @willschlitzer .

File: examples/gallery/symbols/text_symbols.py (hunk 2 above)
Comment (michaelgrund) [reply examples/gallery/symbols/text_symbols.py]: I'm not fully sure, the GMT documentations is not really detailed at this point.

File: examples/gallery/symbols/text_symbols.py [hunk 5]
```diff
@@ -10,10 +10,10 @@
 different characters) and *string* can be a letter or a text string
 (less than 256 characters). Optionally, you can append **+f**\ *font* to
 select a particular font [Default is :gmt-term:`FONT_ANNOT_PRIMARY`] as well as
-**+j**\ *justify* to change the justification [Default is CM]. Outline
-and fill color of the text symbols can be customized via the ``pen``
-and ``color`` parameters, respectively.
-
+**+j**\ *justify* to change the justification [Default is CM]. The outline
+color of the text symbols can be defined via the ``style`` parameter [Default
+is black], the fill color can be set with the ``color`` parameter, and the
```
Comment (seisman) [inline examples/gallery/symbols/text_symbols.py:15]: >  The outline color of the text symbols can be defined via the ``style`` parameter 

It's unclear how to change the outline color via the `style` parameter. Perhaps we should change the previous sentence to

> Optionally, you can append **+f**\ *font*,*outlinecolor* to select a particular font and outline color

File: examples/gallery/symbols/text_symbols.py (hunk 2 above)
Comment (willschlitzer) [reply examples/gallery/symbols/text_symbols.py]: If there won't be another way to change the outline color in the example, I don't think this should say "also"

File: examples/gallery/symbols/text_symbols.py (hunk 5 above)
Comment (michaelgrund) [reply examples/gallery/symbols/text_symbols.py:15]: That's the best solution, done, thanks!

File: examples/gallery/symbols/text_symbols.py (hunk 2 above)
Comment (michaelgrund) [reply examples/gallery/symbols/text_symbols.py]: Done!

**Review by willschlitzer — APPROVED** [review]
Looks good!

**Pull request merged**